suites/pacific/rados/tier-2_rados_test_omap.yaml

```diff
@@ -186,3 +186,21 @@ tests:
             normal_objs: 400
             num_keys_obj: 200001
       desc: Large number of omap creation on objects and OSD resiliency
+  - test:
+      name: Verification of dump scrub parameters
+      desc: Verification of forced and scheduled time flags
+      module: test_scrub_parameters.py
+      polarion-id: CEPH-83593830
+      config:
```

Comment:
Any reason as to why Pools for this test are created with 1 PG count?

Reply:
I am using the pg_id of the pool for the verification, by creating a pool with 1 PG it will be easy to get the PG id of pool in both the replicated and erasure-coded pools.